integration_tests/test_python_plugin.py

```diff
@@ -83,6 +83,16 @@ def test_build_rewrites_shebangs(self):
         self.assertEqual('#!/usr/bin/env python2', python2_shebang)
         self.assertEqual('#!/usr/bin/env python3', python3_shebang)
 
+    def test_build_does_not_keep_pyc_files_in_install(self):
+        project_dir = 'python-entry-point'
```

Comment:
@elopio I think I'll use `pip-requirements-list`, too much copy paste snaps will be bad in the long run.